torch/optim/adam.py

```diff
@@ -140,25 +138,27 @@ def step(self, closure=None):
                 max_exp_avg_sqs,
                 state_steps)
 
-            adam(params_with_grad,
-                 grads,
-                 exp_avgs,
-                 exp_avg_sqs,
-                 max_exp_avg_sqs,
-                 state_steps,
-                 amsgrad=group['amsgrad'],
-                 beta1=beta1,
-                 beta2=beta2,
-                 lr=group['lr'],
-                 weight_decay=group['weight_decay'],
-                 eps=group['eps'],
-                 maximize=group['maximize'],
-                 foreach=group['foreach'],
-                 capturable=group['capturable'],
-                 differentiable=group['differentiable'],
-                 fused=group['fused'],
-                 grad_scale=getattr(self, "grad_scale", None),
-                 found_inf=getattr(self, "found_inf", None))
+            adam(
```

Comment:
This is just style change? Or an actual change?

Reply:
it's the former